### PR TITLE
install clang_osx-64

### LINKED ISSUE #114: emirge not compiling on macOS
I have recently installed emirge on a macOS operating system (macOS Big Sur version 11.2), along with the necessary dependencies. When running the example code (`heat-source-mpi.py`), it displays two platforms I can run it on, and with each platform a different issue arises that breaks the code:
- When choosing`Choice [0]: <pyopencl.Platform 'Portable Computing Language' at 0x######>` the code runs through the first few lines, and then fails with the following exception:
```
ld: library not found for -lm
error: linker command failed with exit code 1
```
- When choosing `Choice [1]: <pyopencl.Platform 'Apple' at 0x######>` the code runs through the same first few lines, and then gives me the option of running on CPU or GPU; regardless of which option I take, however, it throws a new error:
```
KeyError: <pyopencl.Context at 0x#### on <pyopencl.Device '(GPU or CPU)' on 'Apple' at 0x####>>

During handling of the above exception, another exception occurred:
...
<program source>: fatal error: 'pyopencl-complex.h' file not found
    #include <pyopencl-complex.h>
                   ^
(options: -I /path/to/file/)
:
system msg for write_line failure : Bad file descriptor
```
This is strange, as the file (`pyopencl-complex.h`) is present in /path/to/file/, so it should be able to find the file.

I have tried multiple fresh installations, including one after deleting my local anaconda installation, but this issue still persists.

Is it perhaps related to #101 ?

### PR DESCRIPTION
**Pull request by matthiasdiener**
Closes #114 through workaround

### CODE REVIEW

**Comment by matthiasdiener** [comment]
Could you check if this fixes the installation for you @dshtey2 ?

**Comment by dshtey2** [comment]
This fixed the issue for me and everything runs as it should now. The Apple platform is still returning errors finding the file, but I can use the PCL for now with no issue

**Comment by matthiasdiener** [comment]
> This fixed the issue for me and everything runs as it should now. The Apple platform is still returning errors finding the file, but I can use the PCL for now with no issue. Thanks!

Great, thank you!

**Review by dshtey2 — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -138,9 +138,16 @@ if [[ -n "$conda_pkg_file" ]]; then
...
-# manually on Linux.
-[[ $(uname) == "Linux" ]] && conda install pocl-cuda
+# Due to https://github.com/conda/conda/issues/8089, we have to install these
+# packages manually on specific operating systems:
+
+# Required for Nvidia GPU support on Linux (package does not exist on macOS)
+[[ $(uname) == "Linux" ]] && conda install --yes pocl-cuda
+
+# Required to use pocl on macOS Big Sur
+# (https://github.com/illinois-ceesd/emirge/issues/114)
```
Comment (inducer) [inline install.sh:148]:
```suggestion
# https://github.com/illinois-ceesd/emirge/issues/114
# https://github.com/conda-forge/pocl-feedstock/issues/52
```

**Review by inducer — APPROVED** [review]
Sure. Not pretty, but if it helps make it work, then why not?

File: install.sh (hunk 1 above)
Comment (matthiasdiener) [reply install.sh:148]: PR was automerged without this change :-(

**Pull request merged**